packaging/Makefile_packaging.mk

```diff
@@ -125,8 +130,7 @@ all: $(TARGETS)
 	xz -z $<
 
 _topdir/SOURCES/%: % | _topdir/SOURCES/
-	rm -f $@
-	ln $< $@
+	if [ ! -d "$@" ]; then rm -f "$@"; ln "$<" "$@"; fi
```

Comment:
What is $<

Also, isn't $@ a multiple argument thing.   Should this not be a for loop or soemthing

for x in $@; do
# do stuff on $x
done

Reply:
"$@" and "$<" are makefile variables, bash does not see them.